examples/4_simple_storage.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+
+###################################
+# WHAT IS IN THIS EXAMPLE?
```

Comment:
suggest renaming this to `3_simple_storage` so it's clear this is the one people should look at first